crates/ra_syntax/test_data/parser/ok/0066_use_items_trailing_comma.rs

```diff
@@ -0,0 +1 @@
+use ra_syntax::{TextSize, TextRange,};
```

Comment:
While testing this, I accidentally broke the behavior of a `use` tree with a trailing comma. The bug was caught in the IDE tests, but there was no pre-exising parser test so I added one.